### PR TITLE
A few quick quest fixes

### PR DESCRIPTION
**Pull request by chochem**
- fix formatting of redone thaumometric essentia cell quest
- remove loop of prereqs in ae tab that made progressing there impossible
- more small fixes to advanced network tool quest (hide prereq that goes across entire tab and language things)
- various small fixes to new lanth line quests: A lot is just improved grammar and wording. The zpm quest is actually luv. No longer layer the iv and luv option but have them next to each other. Adjust rewards (no reward for just clicking a checkbox, lootbags should match the tier and be usually unenchanted).

### CODE REVIEW

File: config/betterquesting/DefaultQuests/Quests/AppliedEnergisti-AAAAAAAAAAAAAAAAAAAAIw==/NeedMoreInformat-5ObFHI5BQ5Oxv0yca7morA==.json [hunk 1]
```diff
@@ -29,11 +30,11 @@
...
       "lockedProgress:1": 0,
       "autoClaim:1": 0,
       "isSilent:1": 0,
-      "desc:8": "A brand new tools for you amigo.\n\nThis little buddy not only have larger card slot than normal one,\nbut also can show more informations about your network storage.\n\nClick the component ico can switch in ITEM, FLUID, ESSEN detail.\n\n\n§4\nNOTE:\n|  unit  |      range     |\n|    B    |    0~2^10    |\n|   KB   | 2^10~2^20 |\n|   MB   | 2^20~2^30 |\n|   GB   | 2^30~2^40 |\n|   TB   | 2^40~2^50 |\n|   PB   | 2^50~2^60 |\n|   EB   | 2^60~2^70 |\n|   ZB   | 2^70~2^80 |\n|   YB   | 2^80~2^90 |\n|   BB   | over 2^90   |§r"
+      "desc:8": "A brand new tools for you, amigo.\n\nThis little buddy not only has a larger card slot than the normal one, but also can show more information about your network storage.\n\nClicking the component icon can switch between ITEM, FLUID, and ESSENTIA details.\n\n\n§4\nNOTE:\n|  unit  |      range   |\n|    B   |    0~2^10   |\n|   KB   | 2^10~2^20 |\n|   MB   | 2^20~2^30 |\n|   GB   | 2^30~2^40 |\n|   TB   | 2^40~2^50 |\n|   PB   | 2^50~2^60 |\n|   EB   | 2^60~2^70 |\n|   ZB   | 2^70~2^80 |\n|   YB   | 2^80~2^90 |\n|   BB   | over 2^90  |§r"
```
Comment (bombcar) [inline config/betterquesting/DefaultQuests/Quests/AppliedEnergisti-AAAAAAAAAAAAAAAAAAAAIw==/NeedMoreInformat-5ObFHI5BQ5Oxv0yca7morA==.json:37]: should be "A brand new tool"

Comment (chochem) [reply config/betterquesting/DefaultQuests/Quests/AppliedEnergisti-AAAAAAAAAAAAAAAAAAAAIw==/NeedMoreInformat-5ObFHI5BQ5Oxv0yca7morA==.json]: done

File: config/betterquesting/DefaultQuests/Quests/Tier5IV-AAAAAAAAAAAAAAAAAAAABw==/MoreSamarium-u_-8t2IvRnydvmUx_qdYuA==.json [hunk 2]
```diff
@@ -29,7 +29,7 @@
...
+      "desc:8": "If you have a stable Samarium resource acquisition, refining Samarium Ore should be the best way to get Samarium.\n\nAs the first step, disolve some Samarium Ore Concentrate Dust or Crushed Samarium Ore (you probably don\u0027t have access to this option yet) in Nitric Acid in a Digester to get Muddy Samarium Rare Earth Solution. Thorium and Phosphate are retained in the form of a solid slag powder. \n\nThen transfer the muddy solution to a Dissolution Tank and continued to soak in nitric acid. The undissolved powder is then meticulously filtered out, yielding some Cerium.\n\nThen dilute the result with water to adjust the pH."
```
Comment (bombcar) [inline config/betterquesting/DefaultQuests/Quests/Tier5IV-AAAAAAAAAAAAAAAAAAAABw==/MoreSamarium-u_-8t2IvRnydvmUx_qdYuA==.json:32]: "and continued to soak" sounds off - perhaps "and soak it in nitric"

Comment (chochem) [reply config/betterquesting/DefaultQuests/Quests/Tier5IV-AAAAAAAAAAAAAAAAAAAABw==/MoreSamarium-u_-8t2IvRnydvmUx_qdYuA==.json:32]: just a d too much I woul say

**Review by Dream-Master — APPROVED** [review]

**Pull request merged**